app/models/amusement_park.rb

```diff
@@ -0,0 +1,10 @@
+class AmusementPark < ApplicationRecord
```

Comment:
there should be unit tests that assert these validations perform the way in which you expect